uint8 num_module
uint8 sign
float32 lin_vel
float32 ang_vel
float32 delta